LayoutTests/http/tests/performance/performance-resource-timing-entries-default-limit-expected.txt

```diff
@@ -0,0 +1,5 @@
+PASS successfullyParsed is true
+
+TEST COMPLETE
+PASS performance.getEntriesByType("resource").length is 250
```

Comment:
This shouldn't be after `TEST COMPLETE`

Reply:
oh, right. I need to make this an async test.